ga4gh/datamodel/variants.py

```diff
@@ -342,3 +343,118 @@ def getMetadata(self):
         # TODO: Implement this
         ret = []
         return ret
+
+
+def _encode_value(value):
+    if isinstance(value, (list, tuple)):
+        return [str(v) for v in value]
+    else:
+        return [str(value)]
+
+
+_nothing = object()
+
+
+def is_empty_iter(it):
+    """Return True iff the iterator is empty or exhausted"""
+    return next(it, _nothing) is _nothing
```

Comment:
The function is so general that it is likely to be copied into contexts where iteration may include None as a legitimate value.  Thus I create a unique object instance (_nothing) to use a sentinel.